Samples/AppLifecycle/Activation/cpp-console-unpackaged/CppWinRtConsoleActivation.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31229.75
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "CppWinRtConsoleActivation", "CppWinRtConsoleActivation\CppWinRtConsoleActivation.vcxproj", "{F5C337C9-4D9A-465C-9CE2-4B0AD5E411B2}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|x64 = Debug|x64
```

Comment:
Can we add ARM64? (this and all other samples) - note for WPF though, ARM64 isn't supported until .NET 6